topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: tutorial_hands_on
+topic_name: transcriptomics
+tutorial_name: rna-seq-viz-with-cummerbund
+---
+
+# Introduction
+{:.no_toc}
+
+RNA-Seq analysis helps researchers finding new genes and spliced variants, and provides a quantification of cell- and context- specific gene expression. Its data is however complex to handle, and requires both computer science and mathematical knowledge to be managed and interpreted.
```

Comment:
Minor grammar suggestions:

RNA-Seq analysis helps researchers annotate new genes and splice variants, and provides cell- and context-specific quantification of gene expression. RNA-Seq data, however, are complex and require both computer science and mathematical knowledge to be managed and interpreted.